interpreter/code_interpreters/languages/python.py

```diff
@@ -4,6 +4,9 @@
 import re
 
 class Python(SubprocessCodeInterpreter):
+    file_extension = "py"
+    proper_name = "Python"
```

Comment:
These proper names are mostly just being used for the code scanner to say

> Code Scanner: No Issues were found with this Python code

We could just rely on the lowercase language string from the code block, but it felt nicer and more human-readable this way.